src/components/internal/SecuredFields/SecuredFieldsProviderHandlers.ts

```diff
@@ -121,12 +123,23 @@ function handleOnError(cbObj: CbObjOnError, hasUnsupportedCard: boolean = null):
     if (this.state.hasUnsupportedCard && cbObj.fieldType === ENCRYPTED_CARD_NUMBER && hasUnsupportedCard === null) {
         return false;
     }
+    // console.log('### SecuredFieldsProviderHandlers::handleOnError:: cbObj.error', cbObj.error);
```

Comment:
Can we remove this commented console.logs?

Reply:
yup

Reply:
done